packaging/conda/meta.yaml

```diff
@@ -52,18 +52,27 @@ outputs:
         - {{ mpi }}
     files:
       - bin/MODL*
+      - bin/_getprocnumber*
+      - bin/khiops_env # [linux or osx]
+      - bin/khiops # [linux or osx]
+      - bin/khiops_coclustering # [linux or osx]
```

Comment:
I am note sure we should need these for the `khiops-core` Conda package. Idem for the Windows `.cmd`s.

Reply:
Yes, but it offers a very simple way of testing if mpi is well configured (with the 's' flag)

Reply:
IMHO (unless I am missing something), MPI configuration in `khiops_env` should suffice and, for tests, if we source `khiops_env`, then we should be able to just do `MODL -s` . And this would also allow `khiops-python` to be isofunctional with the script by only sourcing `khiops_env`.

Reply:
Ok for keeping these two scripts for debugging purposes.